docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -111,6 +113,22 @@ def __build_simple_minifi_cpp_image_with_root(self):
 
         return self.__build_image(dockerfile)
 
+    def __build_minifi_cpp_image_with_provenance_repo(self):
+        dockerfile = dedent("""FROM {base_image}
+                USER root
```

Comment:
very minor, but this will create a string like this:
```
FROM {base_image}
                USER root
                etc
```
in order to get
```
FROM {base_image}
USER root
etc
```
you need to write
```suggestion
        dockerfile = dedent("""\
                FROM {base_image}
                USER root
```

Reply:
Done, also the other similar ones.